TEST 2, №2/TEST 2, №2.c

```diff
@@ -0,0 +1,95 @@
+﻿#include <stdio.h>
+#include <malloc.h>
+
+
+struct list {
+    int data;
+    struct list* next;
+};
+
+struct list* create(int a) {
+    struct list* root;
+    root = (struct list*)malloc(sizeof(struct list));
```

Comment:
```suggestion
    struct list* root = (struct list*)malloc(sizeof(struct list));
```
Иначе это необъявленная переменная и минус балл за задачу :)